distribution/ddf/pom.xml

```diff
@@ -30,7 +30,7 @@
     <packaging>pom</packaging>
     <properties>
         <distribution.file.name>ddf-${project.version}</distribution.file.name>
-        <opendj-embedded.app.version>1.3.1</opendj-embedded.app.version>
+        <opendj-embedded.app.version>1.3.2-SNAPSHOT</opendj-embedded.app.version>
```

Comment:
Woops, that was actually an accidental commit on my part. Thanks for catching that, haha.